cdc-quickstart-kafka-connect/README.md

```diff
@@ -132,3 +132,12 @@ The docker compose command will bring up the containers for Prometheus and Grafa
     rm -rf zookeeper/zookeeper_data/*
     rm -rf zookeeper/zookeeper_log/*
   ```
+
+* __Unable to deploy connectors locally__
```

Comment:
You can change the heading to be something Mac specific:

__Unable to deploy connectors locally on MacOS__
or
__Docker slows down while working with MacOS__

Reply:
Done.